app/configurator/components/dataset-metadata.tsx

```diff
@@ -68,11 +68,12 @@ export const DataSetMetadata = ({
           </DatasetMetadataTitle>
           <DatasetMetadataBody>
             <Box
+              component="span"
               sx={{ "> a": { color: "grey.900" } }}
               dangerouslySetInnerHTML={{
                 __html: cube.publisher,
               }}
-            ></Box>
+            />
```

Comment:
Nit: At this point I even wonder why we cannot directly pass the sx and dangerouslySetInnerHTML to DatasetMetadataBody.